cmdfactory/builder.go

```diff
@@ -168,15 +169,14 @@ func AttributeFlags(c *cobra.Command, obj any, args ...string) {
 		}
 
 		switch fieldType.Type.Kind() {
-		case reflect.Int:
-			flags.IntVarP((*int)(unsafe.Pointer(v.Addr().Pointer())), name, alias, defInt, usage)
-			flags.Set(name, strValue)
-		case reflect.Int64:
-			flags.IntVarP((*int)(unsafe.Pointer(v.Addr().Pointer())), name, alias, defInt, usage)
```

Comment:
Is the fix here related to the issue? What does this change affect?

Reply:
No, only the last commit is directly related.
This addresses duplicate code around the same location, but the change seems too insignificant for its own PR.

Reply:
I was asking because you also removed the `flags.IntVarP((*...` line, and I was not sure what that changes

Reply:
Did I?

Reply:
Fixed in [4e81a4e9..c2c64c84](https://github.com/unikraft/kraftkit/compare/4e81a4e9f38356b979ec9d9a23103136f772a7c1..c2c64c84924c99244d5228f28413558bb9650654)
Thanks for noticing!